test/main/store/migrations/index.test.js

```diff
@@ -13,6 +14,16 @@ afterAll(() => {
   log.transports.console.level = 'debug'
 })
 
+beforeEach(() => {
+  state = {
+    main: {
+      _version: 0,
+      networks: { ethereum: {} },
+      networksMeta: { ethereum: {} }
+    }
+  }
+})
```

Comment:
If we have this at top level might want to also review the other tests and reset other parts of the state in the same way

Reply:
I think it would be worth it going forward but I quickly tried to go back and retrofit some of the older tests and ran into issues so I decided to just leave them the way they are

Reply:
ehh nm I added a few more and cleaned some up. PR looks a little bloated now but I only just moved some initialization code around